src/WordPress/Blueprints/ResourceManager/Resolver/FilesystemResourceResolver.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace WordPress\Blueprints\ResourceManager\Resolver;
```

Comment:
Why change the namespace from `Resource` to `ResourceManager`? Is `Resource` not allowed in PHP 7.4? If so, let's use `Resources` instead as resolvers are not manager-specific.

Reply:
Also, this PR shows a lot of deleted files and a lot of created files, but I think they are the same files moved to a new location. Check out the `git mv` command – it should make this PR a lot easier to read.

Reply:
Point 1: Resource -> ResourceManager -> Resources
Yeah. It's a soft reserved word. [source](https://www.php.net/manual/en/reserved.other-reserved-words.php) It isn't a must, but it's a tiny ask and we can abide it. Nonetheless, I am changing the name to `Resources` as per your request.

Reply:
Point 2: The change was a tip of the IDE and I deemed it reasonable. But it was done as an suggested action in PHPStorm, so it did its own thing. 

I wonder if it can be configured to use git mv internally... But, that is beyond the point. At this stage - will moving it back and then here again, fix the way it is displayed in this PR?

Reply:
> At this stage - will moving it back and then here again, fix the way it is displayed in this PR?

That is my understanding

Reply:
Sadly, does not work as expected. From what I've read git should detect it as a directory rename, but it is a surprisingly complex git feature. See: [directory-rename-detection](https://git-scm.com/docs/directory-rename-detection)

The scale of the issue is small - can we move past it?

Reply:
> The scale of the issue is small - can we move past it?

Okay, but reluctantly. This just means I need to manually put together a side by side diff to be able to review. It's just five files so I can do it, but let's ensure a proper rename in all future PRs.

Reply:
Will do.

Reply:
@adamziel this regards all the other open threads. I've responded to your inquiry and you gave a thumbs up to my response.

> Point 1: Resource -> ResourceManager -> Resources Yeah. It's a soft reserved word. [source](https://www.php.net/manual/en/reserved.other-reserved-words.php) It isn't a must, but it's a tiny ask and we can abide it. Nonetheless, I am changing the name to `Resources` as per your request.

If we leave $resource the code standards check will fail. You are right on 2 accounts - there is a naming inconsistency now, and this can be done elsewhere. So I'll revert the changes here, but a quick PR with the naming change will be needed to make the pipeline pass the CS step.

Reply:
> > The scale of the issue is small - can we move past it?
> 
> Okay, but reluctantly. This just means I need to manually put together a side by side diff to be able to review. It's just five files so I can do it, but let's ensure a proper rename in all future PRs.

Thanks for that. I've seen you made several commits, but I do not see any display / designation change. If you were able to get forward on that, could you guide me?

Ok. I have an idea what actually happened, correct me if I am wrong. As a mean to mitigate it being a delete/create not a move you moved it back, reviewed the changes within files, and then moved them again. If that is the case, that is really clever.

Reply:
I've just tried to do `git mv` and failed, I think that's because an earlier commit in this branch deletes and recreates those files so they're already somehow 'disconnected' in Git history. Actually, I don't think git tracks any renames internally, [each commit is just a directory structure reflecting new files and dirs](https://www.alibabacloud.com/blog/a-detailed-explanation-of-the-underlying-data-structures-and-principles-of-git_597391). Would that mean the renames are inferred in runtime? 🤔 I don't know, but I'd love to find out in details. Either way, that's out of scope for this discussion.